pkg/network/dhcp/masquerade_test.go

```diff
@@ -85,8 +79,20 @@ var _ = Describe("Masquerade DHCP configurator", func() {
 			return expectedConfig
 		}
 
-		generateExpectedConfigIPv6Enabled := func(vmiSpecNetwork *v1.Network, macString *string, mtu int, ifaceName string, subdomain string) cache.DHCPConfig {
-			expectedConfig := generateExpectedConfigIPv6Disabled(vmiSpecNetwork, macString, mtu, ifaceName, subdomain)
+		generateExpectedConfigOnlyIPv4Enabled := func(vmiSpecNetwork *v1.Network, macString *string, mtu int, ifaceName string, subdomain string) cache.DHCPConfig {
+			expectedConfig := generateExpectedConfig(vmiSpecNetwork, macString, mtu, ifaceName, subdomain)
+			ipv4, _ := netlink.ParseAddr(expectedIpv4)
+			ipv4Gateway, _ := netlink.ParseAddr(expectedIpv4Gateway)
```

Comment:
Cacn we at least assert the errors, in case we badly change expectedIpv4 in the future I we don't catch the error.